src/components/Table/Column.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createElement, forwardRef, ReactNode } from 'react';
+
+/**
+ * Note: typings here could be improved here
+ */
+interface Column {
+  id: string; // ID is required because header is optional
+  header?: string;
+  alignment: 'left' | 'right' | 'center';
+  className?: string;
+}
+
+export interface ColumnProps {
+  alignment: Column['alignment'];
+  className?: string;
+  children: ReactNode;
+}
+
+interface ColumnElementProps {
+  element: 'th' | 'td';
+  children: ReactNode;
+}
+
+export const Header = forwardRef<HTMLTableHeaderCellElement, ColumnProps>(
+  ({ children, ...props }: ColumnProps, ref) => {
+    return (
+      <ColumnElement element="th" {...props} ref={ref}>
+        {children}
+      </ColumnElement>
+    );
+  }
+);
+
+export const Cell = forwardRef<HTMLTableCellElement, ColumnProps>(({ children, ...props }: ColumnProps, ref) => {
+  return (
+    <ColumnElement element="td" {...props} ref={ref}>
+      {children}
+    </ColumnElement>
+  );
+});
+
+const ColumnElement = forwardRef<HTMLTableDataCellElement | HTMLTableHeaderCellElement, ColumnElementProps>(
```

Comment:
Nice - we need to refactor `Button` to use `forwardRef` like this to resolve a console error (I think). Will check that out today by using this as an example